pkg/api/event/nonblocking/user_anonymization_scheduled.go

```diff
@@ -0,0 +1,51 @@
+package nonblocking
+
+import (
+	"github.com/authgear/authgear-server/pkg/api/event"
+	"github.com/authgear/authgear-server/pkg/api/model"
+)
+
+const (
+	UserAnonymizationScheduled event.Type = "user.anonymization_scheduled"
+)
+
+type UserAnonymizationScheduledEventPayload struct {
```

Comment:
Everytime when we add new events, we need to 
1. If it is for event hook, we need to update the hook config (pkg/lib/config/hook.go).
2. If it is for audit log, we need to update the admin API audit log query. Otherwise, the portal audit log page will be broken (visit audit log page from the portal side menu). 

Example: https://github.com/authgear/authgear-server/commit/f05b7a28d0f9945e516056f035b95cd0f19d34d7